packages/atomic/src/components/commerce/Introduction.mdx

```diff
@@ -0,0 +1,167 @@
+          };
+
+          const {context, ...restOfConfiguration} = getSampleCommerceEngineConfiguration();
+
+          let viewUrl = siteConfig[document.title]?.resourceUrl;
+          if (!viewUrl) {
+            console.warn(`No resource URL found for page title "${document.title}". Using fallbackResourceUrl.`);
+            viewUrl = fallbackResourceUrl;
+          }
+
+          const configuration = {
+            context: {
+              ...context,
+              view: {
+                url: viewUrl
+              },
+            },
+            ...restOfConfiguration,
+          };
+
+          await customElements.whenDefined('atomic-commerce-interface');
+          const commerceInterface = document.querySelector('atomic-commerce-interface');
+          await commerceInterface.initialize(configuration);
+
+          commerceInterface.executeFirstRequest();
+        })();
+      </script>
+  </head>
+  <body>
+    <atomic-commerce-interface>
+      <!-- The rest of your component hierarchy -->
+    </atomic-commerce-interface>
+  </body>
+</html>
+```
+
+### Initialization with an engine
+Here is an example of how to initialize an `atomic-commerce-interface` with an engine from `@coveo/headless` .
+The primary difference in the use cases between a standard initialization and using an engine is that an engine allows for a single instance to be shared across multiple interfaces.
+This eliminates boilerplate when initilizating the engine across multiple pages, and it will ensure that all pages have access to the same resources on the engine.
+
+```javascript
+// engine.mjs
+const {
+  buildCommerceEngine, 
+  getSampleCommerceEngineConfiguration
+} = await import('http://static.cloud.coveo.com/headless/v3/commerce/headless.esm.js');
+
+const siteRoot = 'https://www.example.com/';
+const fallbackResourceUrl = `${siteRoot}/search`;
+const siteConfig = {
+  'Example': {href: 'index.html', label: 'Home', resourceUrl: `${siteRoot}/search`},
+};
+
+const {context, ...restOfConfiguration} = getSampleCommerceEngineConfiguration();
+
+let viewUrl = siteConfig[document.title]?.resourceUrl;
+if (!viewUrl) {
+  console.warn(`No resource URL found for page title "${document.title}". Using fallbackResourceUrl.`);
+  viewUrl = fallbackResourceUrl;
+}
+
+export const commerceEngine = buildCommerceEngine({
+  configuration: {
+    context: {
+      ...context,
+      view: {
+        url: viewUrl
+      },
+    },
+    ...restOfConfiguration,
+  },
+});
+```
+
+```html
+<!-- index.html -->
+<html>
+  <head>
+    <title>Example Commerce Interface</title>
+        <script
+            type="module"
+            src="https://static.cloud.coveo.com/atomic/v3/atomic.esm.js">
+        </script>
+        <link
+            rel="stylesheet"
+            href="https://static.cloud.coveo.com/atomic/v3/themes/coveo.css" />
+      <script type="module">
+        (async () => {
```

Comment:
Same as https://github.com/coveo/ui-kit/pull/6692/files#r2604065215, could you remove the async IIFE wrapping.